### DISCUSSION TITLE
How do I run and configure the wiki?

### DISCUSSION THREAD

**Discussion by db0** [django-wiki/django-wiki#1305] [Q&A] [answered]
I may be stupid, but I can't understand how I am supposed to run and configure the wiki

I'm following the instructions here: https://django-wiki.readthedocs.io/en/latest/installation.html

But they just mention the pip installation, and what I think are some environment variables?

I've used pip for the install, but..now what?

```python
python -m pip install wiki pillow --user
```

How do I start it? Do I configure it with a .env file? help please.

**Comment by db0** [comment]
I've now realized I need to merge the django documentation into this install documentation, which is not clear to a newbie like me. I managed to get something running, but it's not the wiki. It would be really good if the installation  instructions are friendly to newbies to django and this wiki. For example a step-by-step on the commands to run and the *complete* sample settings.py to use to get a basic django-wiki running

**Comment by benjaoming** [comment]
Hi @db0 :wave: 

Thanks for the feedback! Not many people take the time to express their docs experience.

I can't really say that I think it's a good idea to provide a FULL setup. There's a number of reasons for that. But it really comes down to this: If you want to use django-wiki, then understanding Django at the level of its introduction tutorial is necessary.

However, I think it's a great idea to give out as much guidance and shortcuts as possible! I've added an intro text here: https://github.com/django-wiki/django-wiki/pull/1306

**Reply by db0** [reply]
Thanks. I've managed to make an end-to-end ansible playbook that can go from a empty debian to a django-wiki. Would you be interested in linking to that for people who want a quick solution to deploy such a wiki?

**Reply by benjaoming** [reply]
That sounds great! I can add a link in the README, LMK :+1:

**Comment by SaadTPTW** [comment, marked as answer]
check this one `https://www.youtube.com/watch?v=wsiHYLW-a1g&t=3s` 👍

**Reply by benjaoming** [reply]
Great link, thanks for sharing! I had no idea that someone had done a video tutorial.

**Reply by benjaoming** [reply]
Okay there's a lot of noise in this tutorial, but it's still great :)
